IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -210,6 +210,7 @@ - (MSIDRefreshToken *)getRefreshableTokenWithAccount:(MSIDAccountIdentifier *)ac
         query.clientId = familyId ? nil : configuration.clientId;
         query.familyId = familyId;
         query.credentialType = credentialType;
+        query.isShared = YES;
```

Comment:
seems a bit dangerous that the caller needs to know if item is shared or not internally. This should be implementation details of the internal cache layer, and is also not applicable to iOS. Can query itself decide if item is shared or not based on the credential type?

Reply:
Done